src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Address/Blocks/form.html.twig

```diff
@@ -72,7 +72,7 @@
           'label': 'Country'|trans({}, 'Admin.Global'),
       }) }}
 
-      <div class="js-manufacturer-address-state">
+      <div class="js-manufacturer-address-state {{ addressForm.children.id_state.vars.attr.class }}">
```

Comment:
Agree with @NeOMakinG 

but please use that:
```suggestion
      <div class="js-manufacturer-address-state{% if addressForm.children.id_state.vars.choices is empty %} d-none{% endif %}">
```

And please remove useless `class` in the Type files.

Reply:
that works. thanks